File: python/18.py
month = int(input("Enter (1-12): "))

match month:
    case 1 | 3 | 5 | 7 | 8 | 10 | 12:
        print("Number of days: 31")
    case 4 | 6 | 9 | 11:
        print("Number of days: 30")
    case 2:
        print("Number of days: 28 or 29")
    case _:
        print("Invalid input")

File: cpp/18.cpp
    #include <iostream>
    using namespace std;

    int main() 
    {
        int x;
        cout << "Enter (1-12): ";
        cin >> x;
        switch (x) 
        {
            case 1: case 3: case 5: case 7:
            case 8: case 10: case 12:
                cout << "Number of days: 31" << endl;
                break;
            case 4: case 6: case 9: case 11:
                cout << "Number of days: 30" << endl;
                break;
            case 2:
                cout << "Number of days: 28 or 29" << endl;
                break;
        }
        return 0;
    }